src/libraries/System.Net.WebSockets.Client/tests/ConnectTest.cs

```diff
@@ -313,5 +315,25 @@ public async Task ConnectAsync_CancellationRequestedAfterConnect_ThrowsOperation
                 catch (IOException) { }
             }, new LoopbackServer.Options { WebSocketEndpoint = true });
         }
+
+        [ConditionalFact(nameof(WebSocketsSupported))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/34690", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
+        [SkipOnPlatform(TestPlatforms.Browser, "CollectHttpResponseDetails not supported on Browser")]
+        public async Task ConnectAsync_Failed()
```

Comment:
I would also argue that the test is not about failed connect async, but about collecting response details. So it might be better to name it accordingly, something like:
```suggestion
        public async Task ConnectAsync_HttpResponseDetailsCollectedOnFailure()
```